dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/SignalN/GeneratorPV.mo

```diff
@@ -22,9 +22,14 @@ model GeneratorPV "Model for generator PV based on SignalN for the frequency han
 
   parameter Types.ReactivePowerPu QMinPu "Minimum reactive power in pu (base SnRef)";
   parameter Types.ReactivePowerPu QMaxPu "Maximum reactive power in pu (base SnRef)";
+  parameter Types.ReactivePower QNomAlt "Nominal reactive power of the generator in Mvar";
```

Comment:
maybe we can specify "considered on alternator side", same everywhere

Reply:
Agreed